Example Apps/Example Swift/ButtonManager.swift

```diff
@@ -105,19 +90,22 @@ private extension ButtonManager {
         }
     }
 
-    /// Returns a soft button that toggles the image visibility state for the SDL UI. The button's text toggles based on the current image visibility.
-    ///
-    /// - Returns: A soft button
-    func softButtonImagesVisible() -> SDLSoftButtonObject {
+    /// Returns a soft button that toggles the image visibility state.
+    private var softButtonImagesVisible: SDLSoftButtonObject {
```

Comment:
Doesn't remove the "battery" icon. It should replace it with text saying "Check Battery"

Reply:
Fixed.